README.md

```diff
@@ -2,52 +2,54 @@
 
 You can deploy 1Password SCIM Bridge on any supported infrastructure that allows ingress by your identity provider and egress to 1Password servers. Here you'll find configuration files and best practices to help with your deployment.
 
-> [!TIP]
-> Before you deploy your SCIM bridge, learn more about [automating provisioning in 1Password using SCIM](https://support.1password.com/scim/).
 
-## Marketplace apps
+### Before you begin
 
-For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean:
+Before you begin deploying 1Password SCIM Bridge, 
+- learn more about [automating provisioning in 1Password using SCIM](https://support.1password.com/scim/).
+- review the [Preparation Guide](/PREPARATION.md). The guide will help you plan for some of the technical components of the deployment and consider some issues you may encounter along the way.
 
-- [Deploy 1Password SCIM Bridge on Google Cloud Platform](https://support.1password.com/scim-deploy-gcp/)
-- [Deploy 1Password SCIM Bridge on DigitalOcean](https://support.1password.com/scim-deploy-digitalocean/)
+## Cloud Platforms
+### Azure
+- [Azure Container Apps (CaaS)](https://support.1password.com/scim-deploy-azure/)
+   - [Container Apps advanced customizations](/azure-container-apps/ADVANCED.md)
 
-## Custom deployment
+- [Azure Kubernetes Service](https://support.1password.com/cs/scim-deploy-azure-kubernetes/)
 
-To customize your deployment, you can use containers as a service or create your own advanced deployment with the examples below as your base.
+### AWS
```

Comment:
Don't forget the `s`!

```suggestion
### Amazon Web Services (AWS)
```
or just

```suggestion
### Amazon Web Services
```

(Do we need the acronym? It will be in the listed deployment methods as well)